src/pages/index.astro

```diff
@@ -57,6 +60,10 @@ const randomFact = TG_FACTS[Math.floor(Math.random() * TG_FACTS.length)];
             I NORGES FETESTE PÅSKEHYTTE!
           </h2>
         </div>
+        <div class="flex flex-col justify-center items-center mb-2">
+          <h2 class="text-white text-3xl font-bold">Lyset av om</h2>
+          <Countdown when={eventTime} />
+        </div>
```

Comment:
Will probably have to do some design/placement tweaks before we can roll anything out on the live front. So excellent if you could skip adding it to front for now.